script.js

```diff
@@ -1,4 +1,57 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "hello world";
+  return myOutputValue;
+};
+var gamemode = "roll die";
+// 0 even for player 1 and odd for player2
+var playerArray = ["Player 1", "Player 2"];
+var chosenArrary = [];
+var die1 = "";
+var die2 = "";
+var counter = 0;
+var chosenNum = "";
+
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var diceNumber = Math.floor(randomDecimal) + 1;
+  return diceNumber;
+};
+
+var main = function (input) {
+  var myOutputValue = "";
+  if (input == "Dice 1") {
+    chosenNum = die1 * 10 + die2;
+    chosenArrary.push(chosenNum);
+    gamemode = "roll die";
+    counter = (counter + 1) % 2;
+    return `Your chosen number is ${chosenNum}.`;
+  } else if (input == "Dice 2") {
+    chosenNum = die2 * 10 + die1;
+    chosenArrary.push(chosenNum);
+    gamemode = "roll die";
+    counter = (counter + 1) % 2;
+    return `Your chosen number is ${chosenNum}.`;
```

Comment:
This block of code could be placed into a helper function and called later on in the main function after the choosing of dice. It would be better if at line 48 you had an `else if(gamemode == "choose die"){ RUN HELPER FUNCTION HERE}`